### PR TITLE
Hw 05

### PR DESCRIPTION
**Pull request by tylerallan**

### CODE REVIEW

**Comment by tylerallan** [comment]
@samruddhis @ncaliskan

**Comment by samruddhis** [comment]
Peer Review of HW-05 
----------------------------------------------------
Topic                       | Excellent | Satisfactory | Needs Work |
----------------------------|-----|-----|-----|
**Coding style**            | :heavy_check_mark: |  |  |
**Coding strategy**         | :heavy_check_mark: |  |  |
**Presentation: graphs**    |  | :heavy_check_mark: |  |
**Presentation: tables**    |  | :heavy_check_mark:  |  |
**Achievement, creativity** | :heavy_check_mark: |  |  |
**Ease of access**          | |  :heavy_check_mark: |  |

Remarks: Overall your code looks great! Just make sure you knit your files so that other people can see your graphs. You also may want to consider adding comments just for easier understanding.

File: Hw_05/hw05.Rmd [hunk 1]
```diff
@@ -56,29 +67,88 @@ Interpret these graphs.
...
+  theme_minimal() +
+  ggtitle("Average Unscented Candle Reviews") +
+  guides(x = guide_axis(n.dodge = 2))
+
+grid.arrange(Scented_Plot, Unscented_Plot, ncol = 2)
 ```
 
+### Description of the Plots
+
+Both plots show a general skew towards the higher end of the scale meaning people are more likely to leave 5 star reviews than 1 star reviews. This trend begins to separate after the cutoff line place on January 20th 2020. Though the trend line for unscented candles does begin to trend down after January 20th, it does not appear to deviate enormously from what would be expected results. The average ratings for scented candles tell a different story. The trend line experiences a large drop off after January 20th 2020. This large decrease in average ratings could be related to individuals not realizing they have lost their sense of smell due to contracting COVID-19.
```
Comment (ncaliskan) [inline Hw_05/hw05.Rmd]: Nice and detailed explanation of the plot!

File: Hw_05/hw05.Rmd [hunk 2]
```diff
@@ -118,12 +188,29 @@ str_no_scent <- function(x){
...
 }
+
+Scented_all$CodeReview <- str_no_scent(Scented_all$Review)
+Scented_all$CodedReview_f <- factor(Scented_all$CodeReview, labels = c("scent", "noscent"))
+print(Scented_all)
 ```
 
 ```{r}
-
+Scented2_Plot <- Scented_all %>%
```
Comment (ncaliskan) [inline Hw_05/hw05.Rmd:139]: You can leave small notes on your code so when you go back to them later on, you will know what you did.

**Review by ncaliskan — COMMENTED** [review]
Nice work!

File: Hw_05/hw05.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+  ggplot() +  aes(x = as.Date(month), y = (..count..)/sum(..count..), color = as.factor(year), fill = as.factor(year)) +
+  geom_bar(alpha = .4) +
+  stat_summary(aes(y = noscent), fun.data = mean_cl_normal) +
+  scale_x_date(name = "Month and Year", date_labels = "%m-%Y", date_breaks = "month", expand = c(0,0)) + 
+  guides(x = guide_axis(n.dodge = 3), color = guide_legend(title = "Year"), fill = guide_legend(title = "Year")) +
+  scale_y_continuous(name = "Percentage of Reviews mentioning 'No Scent'") +
+  labs(title = "Percentage of Scented Candle Reviews Mentioning 'No Scent' in 2019 and 2020")
+
+
+Scented2_Plot
```
Comment (tylerallan) [inline Hw_05/hw05.md:140]: For some reason the plot shown here is from an earlier attempt and does not match the plot that I got when running this code written above.

**Pull request merged**